cluster/cluster.go

```diff
@@ -244,19 +244,17 @@ func CreateTenantService(tenantName string, tenantPort int32, storageClusterNum
 		},
 	}
 
-	res, err := clientset.CoreV1().Services("default").Create(&scSvc);
+	res, err := clientset.CoreV1().Services("default").Create(&scSvc)
 	if err != nil {
 		panic(err.Error())
 	}
-	fmt.Println("done creating tenant service for tenant %s ")
-	fmt.Println(res.String())
+	fmt.Printf("done creating tenant service for tenant %s \n", res.Name)
 
 }
 
-type Tenant struct {
-	Name              string
-	Port              int32
-	StorageClusterNum string
+type SCTenant struct {
+	Name string
+	Port int32
```

Comment:
What is a `Port`?

Reply:
This was a unused struct so I've removed it :P